examples/doxygen/theta_timestepping.cc

```diff
@@ -28,46 +27,47 @@
 using namespace dealii;
 using namespace Algorithms;
 
-
 class Explicit : public OperatorBase
 {
 public:
-  Explicit(const FullMatrix<double> &matrix);
-  void operator() (AnyData &out, const AnyData &in);
+  Explicit(const FullMatrix<double>& matrix);
+  void
+  operator()(AnyData& out, const AnyData& in);
 
 private:
   SmartPointer<const FullMatrix<double>, Explicit> matrix;
-  FullMatrix<double> m;
+  FullMatrix<double>                               m;
```

Comment:
Whether we want this alignment is a point we should discuss. Generally I am not a big fan of it.

Reply:
I like the aligned function argument names.

Reply:
@bangerth 

> aligned function argument

that's not function argument as far as I can see, it's class members. But I do like them aligned nonetheless.

Reply:
That's what I meant. My fingers just didn't type it that way...

Reply:
The alternative is https://github.com/masterleinad/dealii/pull/11.